src/EFTools/DesignXmlCore/DesignXmlCore.csproj

```diff
@@ -63,16 +63,9 @@
     <Reference Include="Microsoft.VisualStudio.Shell.$(VisualStudioVersion15)" />
     <Reference Condition="'$(VisualStudioVersion)' == '16.0'" Include="Microsoft.VisualStudio.Shell.Framework" />
     <Reference Include="Microsoft.VisualStudio.Package.LanguageService.$(VisualStudioVersion15)" />
-    <Reference Include="Microsoft.VisualStudio.OLE.Interop" />
+    <Reference Include="Microsoft.VisualStudio.Interop" />
```

Comment:
❔ This should be `PackageReference` right?

Reply:
This repo is using the old-school project system with `packages.config`, so `PackageReference` isn't recognized.

Reply:
After much pain and many iterations I ended up moving this to `PackageReference` anyway